renderdoc/os/os_specific.h

```diff
@@ -155,6 +155,13 @@ struct Semaphore
 protected:
   Semaphore();
   ~Semaphore();
+
+private:
+#if !defined(__APPLE__) && !defined(_WIN32) && !defined(__linux__)
```

Comment:
Please do not add new declarations and preprocessor conditionals like this into os_specific.h